test/fixtures/source-map/sigint.js

```diff
@@ -0,0 +1,8 @@
+const a = 99;
+if (true) {
+  const b = 101;
+} else {
+  const c = 102;
+}
+process.kill(process.pid, "SIGINT");
+//# sourceMappingURL=https://http.cat/402
```

Comment:
I love these - but I would still prefer it if we did not have domains we do not control here since this can turn into a malicious domain at some point.

Reply:
fair point, just use something we control like nodejs.org, for test?

Reply:
Yeah